backend/tests/modules/user/antisybil/test_antisybil.py

```diff
@@ -43,6 +43,41 @@ def test_antisybil_service(
     assert score == 2.572
 
 
+def test_guest_stamp(
+    patch_gitcoin_passport_issue_address_for_scoring,
+    patch_gitcoin_passport_fetch_score,
+    patch_gitcoin_passport_fetch_stamps,
+    mock_users_db,
+):
+    context = get_context(4)
+
+    service = GitcoinPassportAntisybil()
+    alice, _, _ = mock_users_db
+
+    score, expires_at, stamps = service.fetch_antisybil_status(context, alice.address)
+    service.update_antisybil_status(context, alice.address, score, expires_at, stamps)
+    score, _ = service.get_antisybil_status(context, alice.address)
+    assert score == 2.572  # guest list score bonus not applied
+
+    guest_address = "0x2f51E78ff8aeC6A941C4CEeeb26B4A1f03737c50"
+    database.user.add_user(guest_address)
+    score, expires_at, stamps = service.fetch_antisybil_status(context, guest_address)
+    service.update_antisybil_status(context, guest_address, score, expires_at, stamps)
+    score, _ = service.get_antisybil_status(context, guest_address)
+    assert (not stamps) and (
+        score == 21.0
+    )  # is on guest list, no stamps, applying 21 score bonus manually
+
+    stamp_address = "0xBc6d82D8d6632938394905Bb0217Ad9c673015d1"
+    database.user.add_user(stamp_address)
+    score, expires_at, stamps = service.fetch_antisybil_status(context, stamp_address)
+    service.update_antisybil_status(context, stamp_address, score, expires_at, stamps)
+    score, _ = service.get_antisybil_status(context, stamp_address)
+    assert (stamps) and (
+        score == 22.0
+    )  # is on guest list, HAS GUEST LIST STAMP, score is from fetch
```

Comment:
i don't think we should call external API here in our unit testing due to a few reasons:
1) overhead from sending many potential requests in UTs
2) GUEST LIST may change on GC API and then we get flaky UTs